imod/mf6/multimodel/exchange_creator_unstructured.py

```diff
@@ -57,10 +57,23 @@ def _compute_geometric_information(self) -> pd.DataFrame:
         grid = self._submodel_labels.ugrid.grid
         edge_face_connectivity = grid.edge_face_connectivity
 
-        face1, face2 = edge_face_connectivity[self._connected_cell_edge_indices].T
+        unordered_face1, unordered_face2 = edge_face_connectivity[
+            self._connected_cell_edge_indices
+        ].T
+        face_partition_1 = self._get_partition_numbers(unordered_face1)
+        face_partition_2 = self._get_partition_numbers(unordered_face2)
+        face1 = np.where(
+            face_partition_1 > face_partition_2, unordered_face2, unordered_face1
+        )
+        face2 = np.where(
```

Comment:
I had to do some thinking about why the order of the faces matters, i.e. why does it matter what is face1 and what is face2? After reading the code hereafter and thinking about it, I guess it's because in the exchanges there is a first and second partition too? And in that case, you'd want the orientation to point in the right direction?

At any rate, it's not obvious to me, so it would be nice to write some comments here.

Reply:
it is indeed as you say: we want the exchange from the partition with the lower number to that with the higher number. I added a comment to that effect.